core/assets/bundles/bundle_de.properties

```diff
@@ -747,12 +753,12 @@ mode.attack.description = Keine Wellen, das Ziel ist es die gegnerische Basis zu
 mode.custom = Angepasste Regeln
 
 rules.infiniteresources = Unbegrenzte Ressourcen
-rules.reactorexplosions = Reactor Explosions
+rules.reactorexplosions = Kraftwerke explodieren
```

Comment:
Nicht "Kraftwerkexplosionen"?

Reply:
Name the situation (that are ..)  = Kraftwerksexplosionen
Tell some on what happen = Kraftwerke explodieren
May better (possibility) = Kraftwerke könnten explodieren